cli/tests/fixtures/poetry.lock

```diff
@@ -0,0 +1,1123 @@
+[[package]]
```

Comment:
Not sure if there's any recommendation on what kind of input files are usually used for tests. I just copied the `poetry.lock` from `poetry` itself. However this is kinda big and the density of interesting edgecases is probably not particularly high.

Reply:
Not sure either...but this file should work for now. It can be changed later, as we learn more about interesting edge cases.